src/annotator/anchoring/test/text-range-test.js

```diff
@@ -88,6 +88,38 @@ describe('annotator/anchoring/text-range', () => {
       });
     });
 
+    describe('#relativeTo', () => {
+      it("throws an error if argument is not an ancestor of position's element", () => {
+        const el = document.createElement('div');
+        el.append('One');
+        const pos = TextPosition.fromPoint(el.firstChild, 0);
+
+        assert.throws(() => {
+          pos.relativeTo(document.body);
+        }, 'Parent is not an ancestor of current element');
+      });
+
+      it('returns a TextPosition with offset relative to the given parent', () => {
+        const grandparent = document.createElement('div');
+        const parent = document.createElement('div');
+        const child = document.createElement('span');
+
+        grandparent.append('a', parent);
+        parent.append('bc', child);
+        child.append('def');
+
+        const childPos = TextPosition.fromPoint(child.firstChild, 3);
+
+        const parentPos = childPos.relativeTo(parent);
+        assert.equal(parentPos.element, parent);
+        assert.equal(parentPos.offset, 5);
```

Comment:
Because of `#text 'bc'` in `parent`, yes?

Reply:
Yes